FullEvents/NavigationViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  NavigationViewController.swift
+//  FullEvents
+//
+//  Created by user on 10/10/17.
+//  Copyright © 2017 user. All rights reserved.
+//
+
+import UIKit
+
+class NavigationViewController: UINavigationController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        
+        let token = UserDefaults.standard.string(forKey: "accessToken")
+        
+        guard let accessToken = token, !accessToken.isEmpty else {
+            
+            print("User information not available, please go into loginpage.")
+            
+            return
+        }
+        
+        
+        print("AccessToken is available land into Dashboard \(accessToken)")
```

Comment:
Always set navigation controller or landing page as root view controller in Storyboard.